store/store_download.go

```diff
@@ -55,6 +56,11 @@ var downloadRetryStrategy = retry.LimitCount(7, retry.LimitTime(90*time.Second,
 	},
 ))
 
+var downloadSpeedMeasureWindow = 5 * time.Second
```

Comment:
My gut feeling (but it's not more than that!) is that 5s is way too low, I would go with something like 1min-5min or something similar. I imagine that there will be times on cellular networks or other unreliable links where there is simple no network connectivity for several seconds until the signal is ok again. **But** that is just my educated guess, probably best to talk to field about this.

Reply:
Yes, I think you're right. I think *minutes* are perfectly fine.

Reply:
Updated to 5 minutes for now.